cfgov/jobmanager/models/pages.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import absolute_import
```

Comment:
No, there is no more `jobmanager.models.py`. It's been converted to a package, with everything under `jobamanager/models`. The things that were formerly in `models.py` (Django models) are now in `jobmanager.models.django`.